gp-includes/formats/format_pomo.php

```diff
@@ -31,11 +31,37 @@ public function print_exported_file( $project, $locale, $translation_set, $entri
 		foreach( $entries as $entry ) {
 			$po->add_entry( $entry );
 		}
-		$po->set_header( 'Project-Id-Version', $project->name );
+
+		$current = $project;
+		$project_tree = array();
+		$project_tree[] = $current->name;
+
+		while ( $current->parent_project_id > 0 ) {
+			$current = GP::$project->get( $project->parent_project_id );
+			$project_tree[] = $current->name;
+		}
+
+		$project_tree = array_reverse( $project_tree );
+
+		$project_id_version = implode( ' - ', $project_tree );
+
+		/**
+		 * Filter the project name before export.
```

Comment:
This should mention `Project-Id-Version` as it's not only the project name.

From https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html
> **Project-Id-Version**
> This is the name and version of the package. Fill it in if it has not already been filled in by `xgettext`.